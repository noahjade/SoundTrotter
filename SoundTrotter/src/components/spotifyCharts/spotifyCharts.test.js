import { shallow } from "enzyme";
import SpotifyCharts from ".";

test("Correctly renders default spotify charts component", () => {
    let selectedCountry = {
        code: "NONE",
        name: "NONE",
        artists: []
    }
    let topSongs = [];
    const defaultSpotifyChartsComponent = shallow(<SpotifyCharts topSongs={topSongs} selectedCountry={selectedCountry} />);
    expect(defaultSpotifyChartsComponent).toMatchSnapshot();
});