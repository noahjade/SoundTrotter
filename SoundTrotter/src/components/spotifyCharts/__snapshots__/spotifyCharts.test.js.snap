// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders default spotify charts component 1`] = `
<div
  className="container"
>
  <div
    className="title"
  >
    Top 10 songs in 
    ...
  </div>
  <div
    className="message"
  >
    Select a country
  </div>
</div>
`;
